examples/jsm/loaders/SVGLoader.js

```diff
@@ -69,37 +69,37 @@ SVGLoader.prototype = {
 
 				case 'path':
 					style = parseStyle( node, style );
-					if ( node.hasAttribute( 'd' ) ) path = parsePathNode( node, style );
+					if ( node.hasAttribute( 'd' ) ) path = parsePathNode( node );
```

Comment:
see https://github.com/mrdoob/three.js/pull/16572#discussion_r287752468

Reply:
just fixed the js file too and rerun the modularize script